src/Lykke.Job.LiteCoin/PeriodicalHandlers/UpdateBalancesPeriodicalHandler.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Common;
+using Common.Log;
+using Lykke.Job.LiteCoin.Functions;
+
+namespace Lykke.Job.LiteCoin.PeriodicalHandlers
+{
+    public class UpdateBalancesPeriodicalHandler:TimerPeriod
```

Comment:
Consider to use TimerTrigger over aggregation instead of TimerPeriod over inheritance

Reply:
We can't use timer TimerTriggerAttribute here, because we need to configer timeout through settings - TimerTriggerAttribute allows to  use constants only.